jenkins_jobs/gpii-couchdb-test-harness.yml

```diff
@@ -68,7 +68,7 @@
     node: h-0005.tor1.incd.ca
     workspace: $parent_workspace
     builders:
-      - shell: DISPLAY=:0 BUILD_ID=gpii-binder vagrant ci test linux
+      - shell: DISPLAY=:0 BUILD_ID=gpii-couchdb-test-harness vagrant ci test linux
```

Comment:
Is this change right? Surely it should mention gpii-webdriver rather than gpii-couchdb-test-harness

Reply:
Ah sorry, I see the full diff now

Reply:
Yes, this was a goof from the last pull.